### DISCUSSION TITLE
Today's Plex Media Server update - Agents 'fix' (2023-08-03)

### DISCUSSION THREAD

**Discussion by jfreiman** [CodyBerenson/PGMA-Modernized#270] [General]
Is this going to help PGMA retrieve results better or undue fixes that you made to PGMA?

Version 1.32.6.7371
Fixes
(Agents) Some attributes were missing from legacy agent api responses (#14436)

**Comment by JPH71** [comment]
It fixes the 403 error...
And corrections have been put in raised by the issues...

Just working some more on the reporting on 403 errors whether they are
raised on the film or cast or director

This will be shows in the film legend...

**Comment by JPH71** [comment]
Send me a list of the introduced errors...

**Comment by jfreiman** [comment]
I knew you'd be on top of it!
Thanks for your continued hard work.

Sent from from an Ultra phone on T-Mobile's 5G network
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by JPH71** [comment]
Xxxx

**Comment by CodyBerenson** [comment]
@jfreiman  Hi John, I'm having trouble following you here, as we generally try to track folks issues and requirements requests. What specifically are you asking for that you knew that Jason would" be on top of?"  It isn't clear what you mean by "retrieve results better". If you have specific thoughts for improvements or specific issues to report please do.  The release and the  notes that accompany 1) describe whats changed and 2) link to any specific issues or enhancements that were open (in this case, resolving the 403 issue that has been plaguing the community as well as updating the thumbor address that had just expired. Cheers.

**Comment by jfreiman** [comment]
I put this in the Discussion/General part of the forum, not the Issues section.

I was just curious if this change/reversion on the PMS would effect/improve PGMA -- and as you stated, it fixed an issue which you otherwise already fixed.

That's all.  I'm not asking for anything new.
